db/db.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"fmt"
+	"database/sql"
+//	"log"
+	"os"
+	_ "github.com/lib/pq"
+)
+
+
+func Dquery(query string) (*sql.DB, *sql.Rows, error) {
+	host := os.Getenv("DB_HOST")
+	port := os.Getenv("DB_PORT")
+	user := os.Getenv("DB_USER")
+	password := os.Getenv("DB_PASS")
+	dbname := os.Getenv("DB_DB")
+
+	con := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s sslmode=disable", host, port, user, password, dbname)
+
+	db, err := sql.Open("postgres", con)
```

Comment:
This will end up leaking connections to postgres, and I'll attach a suggested change below that uses defer to make sure we close this before the function ends.